man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -245,6 +250,49 @@ piadm(1M) -- Manage SmartOS Platform Images
  [root@smartos ~]#
 ```
 
+### Customizing a boot image to load /etc/ppt_aliasses and /etc/ppt_matches
+
+```
+ [root@smartos ~]# piadm list
+ PI STAMP           BOOTABLE FILESYSTEM            BOOT IMAGE   NOW   NEXT
+ 20200714T195617Z   standalone/boot                next         yes   yes
+ [root@smartos ~]# mkdir -p /standalone/boot/common
```

Comment:
I'm gonna stop you right there.  I wonder if we should be doing this as part of `piadm bootable -e` or `piadm bootable -r`.  Let's not push that on the user, per se (but yes, you've documented it well above).

Reply:
I don't really like this section and I think it should be entirely removed.

These config files are how to control the behavior of *loader*. While loading modules is supported, it's not intended to be a general use tool kit for overriding portions of the platform image.

The example of modifying these particular files, IMO is overly specific, and in general is something that should be avoided unless you *really* know what you're doing. There are a very few users who use this frequently, but the vast majority of users will have no need. Putting this right in the man page here, to me, makes it seem like this is a common, or recommended practice, that most users will need. When in reality almost all users should not be modifying these files.

Reply:
I agree with the KISS approach, I will switch to the paragraph suggested here https://github.com/joyent/smartos-live/pull/958#discussion_r495319106.

That just leaves open if we should create `common/` on bootable and if we should update all existing images or not, I'm leaning on maybe on the first and no for the later.